### PR TITLE
Update codegen version and generate auth file

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:
This generates `service.pb.auth.go` file using the new codegen version.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by knanao** [comment]
@nghialv 
Sorry, Please add `docker://ghcr.io/pipe-cd/codegen@sha256:9ccd18fb8976fb3ff206755a32ae79f0d4f42a444e6a30024db4d35313db30fa` temporally.

**Comment by nghialv** [comment]
> @nghialv Sorry, Please add `docker://ghcr.io/pipe-cd/codegen@sha256:9ccd18fb8976fb3ff206755a32ae79f0d4f42a444e6a30024db4d35313db30fa` into whitelist temporally.

Done!

**Review by khanhtc1202 — APPROVED** [review]
👍

File: pkg/app/server/service/webservice/service.pb.auth.go [hunk 1]
```diff
@@ -18,116 +18,173 @@
...
-	case "/grpc.service.webservice.WebService/EnableStaticAdmin":
-		return isAdmin(r)
-	case "/grpc.service.webservice.WebService/DisableStaticAdmin":
-		return isAdmin(r)
-	case "/grpc.service.webservice.WebService/UpdateProjectSSOConfig":
-		return isAdmin(r)
-	case "/grpc.service.webservice.WebService/UpdateProjectRBACConfig":
-		return isAdmin(r)
+	case "/grpc.service.webservice.WebService/GetCommand":
+		return true
```
Comment (nghialv) [inline pkg/app/server/service/webservice/service.pb.auth.go:105]: 👍

**Review by nghialv — APPROVED** [review]
👍

**Review by khanhtc1202 — APPROVED** [review]
🙌

**Pull request merged**